build.gradle

```diff
@@ -11,7 +11,7 @@ buildscript {
 
     dependencies {
         classpath "gradle.plugin.com.github.maiflai:gradle-scalatest:0.25"
-        classpath "cz.alenkacz:gradle-scalafmt:1.3.5"
```

Comment:
I only bumped the dependency, assuming the project should still use the default config.
I'm not aware of any conventions is Scala language, I'm not sure if I ever wrote a single line using it 😛

Reply:
it's ok :) don't worry